mtg_draft_ai/deckbuild.py

```diff
@@ -1,12 +1,50 @@
 """Deckbuilding functions."""
 
 from mtg_draft_ai import synergy
+import networkx as nx
 
 
-_NONLANDS_IN_DECK = 23
+_NONLANDS_IN_DECK_DEFAULT = 23
 
+_COLOR_COMBOS = ['WU', 'WB', 'WR', 'WG', 'UB', 'UR', 'UG', 'BR', 'BG', 'RG']
```

Comment:
What is the reason for only including dual color combos?

Reply:
Well the bots only try to draft 2-color combos and don't know what fixing is yet so... 🤷‍♀ 

I think fixing and 3-color won't be too difficult but we'll need to do it first.

Reply:
I see, so the color system logic is effectively living in multiple places currently.